tests/markdown_table/jsfmt.spec.js

```diff
@@ -1 +1,7 @@
+const path = require("path");
+
 run_spec(__dirname, ["markdown"], { proseWrap: "always" });
+
+run_spec(path.join(__dirname, "long"), ["markdown"], {
+  proseWrap: "never"
```

Comment:
`run_spec` is supposed to be always used with `__dirname`, you can add a `jsfmt.spec.js` in `tests/markdown_table/long/` to achieve this goal.